src/testing/puppeteer/puppeteer-screenshot.ts

```diff
@@ -137,7 +137,7 @@ export async function pageCompareScreenshot(
   return results;
 }
 
-function createPuppeteerScreenshotOptions(opts: ScreenshotOptions) {
+export function createPuppeteerScreenshotOptions(opts: ScreenshotOptions, width: number, height: number) {
```

Comment:
Can we adjust this function a bit so that it's clear that `width` and `height` are meant to be provided together? I know its a bit nitpicky, but I think that it'll help to demonstrate that these two value are related to one another:
```suggestion
export function createPuppeteerScreenshotOptions(opts: ScreenshotOptions, {width, height}: {width: number, height: number}) {
```